llvm/lib/Target/NVPTX/NVPTXISelLowering.cpp

```diff
@@ -149,12 +150,13 @@ static bool IsPTXVectorType(MVT VT) {
   }
 }
 
-static bool Isv2f16Orv2bf16Type(EVT VT) {
-  return (VT == MVT::v2f16 || VT == MVT::v2bf16);
+static bool Isv2f16Orv2bf16Orv2i16Type(EVT VT) {
```

Comment:
Ah, we already have that helper. Move it upwards and use it for the comparisons above.

Maybe rename it to something less cumbersome. `Isv2x16Type` ?

Reply:
done.